pkg/scalers/rabbitmq_scaler.go

```diff
@@ -21,6 +21,12 @@ import (
 	kedautil "github.com/kedacore/keda/v2/pkg/util"
 )
 
+var anonymizePattern *regexp.Regexp
```

Comment:
```suggestion
var rabbitMQAnonymizePattern *regexp.Regexp
```
Nit, since this is set on a package level.

Reply:
you're right :)